docs/source-pytorch/visualize/supported_exp_managers.rst

```diff
@@ -8,7 +8,8 @@ To use `Comet.ml <https://www.comet.ml/site/>`_ first install the comet package:
 
 Configure the logger and pass it to the :class:`~pytorch_lightning.trainer.trainer.Trainer`:
 
-.. code-block:: python
+.. testcode::
```

Comment:
Why this when we are dropping loggers, so they are never installed?

Reply:
Docs build was failing. Making it a testcode means we can use skipif

Reply:
so far I know `.. code-block:: python` are not tested...

Reply:
I think so. Maybe I misjudged where the error was coming from. If you want, I can revert the testcode in a follow-up